lib/libzpool/vdev_disk_aio.c

```diff
@@ -404,6 +408,95 @@ kick_submitter(vdev_disk_aio_t *vda)
 	assert(rc == sizeof (data));
 }
 
+#define	SCSI_STATUS_GOOD	0
+#define	SCSI_STATUS_CHECK_COND	2
```

Comment:
should we get these constants from some standard header files?

Reply:
any standard header files with these constants @jkryl ?

Reply:
Thanks Vitta! I found most of them defined in scsi/scsi.h linux header file. Only two defines remained and those have been moved to the beginning of the file to usual place.